src/main/java/com/plugish/woominecraft/WooMinecraft.java

```diff
@@ -285,10 +293,16 @@ private String getPendingOrders() throws Exception {
 				throw new FileNotFoundException(e.toString());
 			}
 		} catch (FileNotFoundException e) {
+			try {
+				in = new BufferedReader(new InputStreamReader(baseURL2.openStream()));
+			} catch (IOException t) {
+				e.getMessage();
+			}
 			String key = getConfig().getString("key");
-			return e.getMessage().replace( key == null ? "" : key, "privateKey");
+			e.getMessage().replace( key == null ? "" : key, "privateKey");

```

Comment:
Again, we're doing nothing with the returned value here it'll just sit in limbo.